internal/util/k8sutil/k8sutil.go

```diff
@@ -102,7 +106,11 @@ func FormatOperatorNameDNS1123(name string) string {
 // by removing characters from the beginning of label such that len(label) <= 63.
 func TrimDNS1123Label(label string) string {
 	if len(label) > validation.DNS1123LabelMaxLength {
-		return label[len(label)-validation.DNS1123LabelMaxLength:]
+		lbl := label[len(label)-validation.DNS1123LabelMaxLength:]
+		if len(validation.IsDNS1123Label(lbl)) != 0 {
+			return FormatOperatorNameDNS1123(lbl)
+		}
+		return lbl
```

Comment:
```suggestion
		return strings.TrimSpace(label[len(label)-validation.DNS1123LabelMaxLength:], "-")
```

Reply:
`TrimSpace` only takes one argument. This seems to work `return strings.Trim(label[len(label)-validation.DNS1123LabelMaxLength:], "-")`